### ISSUE TITLE
leveling

### ISSUE THREAD

**Issue by hkubus** [Xiboon/Alcan#42]

**Comment by Nyameliaaaa** [comment]
```js
((5 / 3) * (x ** 3)) + ((45 / 2) * (x ** 2)) + ((455 / 6) * x)
```

mee6 level formula, where x is the number of levels

**Comment by hkubus** [comment]
i hate it

**Comment by hkubus** [comment]
I love it

**Comment by SpreeHertz** [comment]
[discord-xp](https://www.npmjs.com/package/discord-xp) may make your work easier.

**Comment by SuperchupuDev** [comment]
@SpreeHertz i don't think we can use it because the lib uses mongoose, we use prisma on top of postgres

**Comment by Nyameliaaaa** [comment]
> @SpreeHertz i don't think we can use it because the lib uses mongoose, we use prisma on top of postgres

That doesn't mean we can't just reverse engineer it and create LevelingModule.ts

**Comment by SuperchupuDev** [comment]
or we can just write it ourselves™

**Comment by hkubus** [comment]
reverse engineer deez nuts
